Database/Create_timeseriesmetadata_table.sql

```diff
@@ -0,0 +1,18 @@
+create table `sdliticadb`.`timeseriesmetadata` (
```

Comment:
`Server version: 10.2.31-MariaDB SUSE package`

Gives me this:
`ERROR 1005 (HY000): Can't create table `sdliticadb`.`timeseriesmetadata` (errno: 150 "Foreign key constraint is incorrectly formed")`